### ISSUE TITLE
Trailing whitespace in `IDENTITY COLUMN` and `NOT VALID` before semicolon

### ISSUE THREAD

**Issue by bolajiwahab** [lelit/pglast#165]
Trailing whitespace in ` IDENTITY COLUMN` and `NOT VALID` constraints.
```sql
echo -e """    ALTER TABLE color 
        ALTER COLUMN color_id ADD GENERATED BY DEFAULT AS IDENTITY;
""" | pgpp -C -e

ALTER TABLE color ALTER COLUMN color_id ADD GENERATED BY DEFAULT AS IDENTITY ;
```

```sql
echo -e """ALTER TABLE books     
        ADD CONSTRAINT distfk
            FOREIGN KEY (author_id) REFERENCES authors (author_id) ON UPDATE RESTRICT NOT VALID;
""" | pgpp -C -e

ALTER TABLE books ADD CONSTRAINT distfk FOREIGN KEY (author_id) REFERENCES authors (author_id) ON UPDATE RESTRICT NOT VALID ;
```
Is this the expectation or likely a bug?

**Comment by lelit** [comment]
Thanks for spotting and reporting it, I will take care of that as time permits.

**Comment by bolajiwahab** [comment]
I can also take care of it if that is fine.

**Comment by lelit** [comment]
Oh, thanks a lot, but I'm almost done, while waiting for the coffee this morning... I will commit this evening.

**Comment by lelit** [comment]
This is fixed in just released v6.11 and v7.1.
